doc/intro.md

```diff
@@ -84,6 +81,23 @@ Three alternatives are available to produce population projections:
 
 This method involves projecting births, deaths and migrations by age and sex.
 
+See below the minimal cohort component method we aim to provide:
+
+![minimal CCM](images/MVP_generic_CCM_flowchart.png)
+
+Note:
+
+We will ultimately provide four alternatives for projecting the components of change (i.e. fertiliy, mortality and migration): using fixed values / rates or using trend values / rates.
```

Comment:
Change to:
We will ultimately provide four alternatives for projecting the components of change (i.e. fertiliy, mortality and migration): projected values or rates can either be fixed, or determined by applying a national trend.